src/library.js

```diff
@@ -1477,6 +1477,8 @@ LibraryManager.library = {
   llvm_trunc_f64: 'Math_trunc',
   llvm_floor_f32: 'Math_floor',
   llvm_floor_f64: 'Math_floor',
+  llvm_round_f64: 'Math_round',
```

Comment:
please change the order here, to put the f64 one first (to be consistent with the ones before)